CoreWebServer/src/au/gov/asd/tac/constellation/webserver/api/ConstellationApiServlet.java

```diff
@@ -25,11 +27,21 @@
 /**
  * A web service which makes up part of the Constellation REST API. The use of a
  * secret is enforced for this type of web service.
+ * <p>
+ * Any exceptions thrown while executing a servlet call are caught and converted
+ * to an HttpServletResponse.sendError() response, as well as being logged
+ * at Level.INFO (to avoid an error dialog box being displayed).
```

Comment:
@algol60 I'm going to suggest we log at to `Logger.ERROR` instead. It will still appear on the output console but will be prefixed with `[ERROR]`. I'm going to merge your PR and send through a follow up PR with this minor suggestion rather than hold up your PR any longer.